RockWeb/Blocks/Crm/PersonDetail/EditGroup.ascx.cs

```diff
@@ -1246,20 +1229,20 @@ protected void btnSave_Click( object sender, EventArgs e )
                                     }
                                 }
                             }
+
```

Comment:
With the code above removed, this code did not need to be run in all instances. I'll be putting this back as it was as well, thanks for the review.